src/jsMain/kotlin/team/karakum/showcase/material/utils/ShowcaseBox.kt

```diff
@@ -0,0 +1,21 @@
+package team.karakum.showcase.material.utils
+
+import mui.material.Box
+import mui.system.sx
+import react.FC
+import react.PropsWithChildren
+import web.cssom.Display
+import web.cssom.JustifyContent
+import web.cssom.em
+
+val ShowcaseBox = FC<PropsWithChildren> { props ->
+    Box {
+        sx {
+            display = Display.flex
+            justifyContent = JustifyContent.left
+            gap = 1.em
```

Comment:
Why `em`?

Reply:
looks responsive enough and dependent on user's screen, but we can do pixels